index.html

```diff
@@ -0,0 +1,181 @@
+            <li>Back End Dev</li>
+            <li><img src="./image/image4.png" alt="count2"></li>
+            <li>2015</li>
+        </ul>
+        <p>
+            A daily selecion of privately personalized reads; no accounts or sign up required.
+        </p>
+         <ul class="lang">
+            <li><h5>html</h5></li>
+            <li><h5>css</h5></li>
+            <li><h5>javascript</h5></li>
+         </ul>
+         <button type="button">See Project</button>
+         </div>
+        <div class="card3">
+            <img src="./image/card3.png" alt="">
+        <h3>TONIC</h3> 
+          <ul>
+           <li>CANOPY</li>
+         <li><img src="./image/image3.png" alt="count1"></li>
+            <li>Back End Dev</li>
+            <li><img src="./image/image4.png" alt="count2"></li>
+            <li>2015</li>
+        </ul>
+        <p>
+            A daily selecion of privately personalized reads; no accounts or sign up required.
+        </p>
+         <ul class="lang">
+            <li><h5>html</h5></li>
+            <li><h5>css</h5></li>
+            <li><h5>javascript</h5></li>
+         </ul>
+         <button type="button">See Project</button>
+        </div>        
+        <div class="card4">
+            <img src="./image/card4.png" alt="">
+        <h2>Multi-Post <br>Stories</h2> 
+          <ul>
+           <li>CANOPY</li>
+         <li><img src="./image/image4.png" alt="count1"></li>
+            <li>Back End Dev</li>
+            <li><img src="./image/image4.png" alt="count2"></li>
+            <li>2015</li>
+        </ul>
+        <p>
+            A daily selecion of privately personalized reads; no accounts or sign up required.
+        </p>
+         <ul class="lang">
+            <li><h5>html</h5></li>
+            <li><h5>css</h5></li>
+            <li><h5>javascript</h5></li>
+         </ul>
+         <button type="button">See Project</button>
+        </div>
+    
+    </section>
+    <section>
+    
+    <div class="about">
+        <h2>About <br> MySelf</h2>
+        <p class="intro">
+            Hello I am a software developer! I can help you build a product ,feature or website Look through some of my work and experience! if you like what you see and have a project you need coded, don't hesitate to contact me.
+        </p>
+        <div class="contact">
+            <h3>LET'S  CONNECT</h3>
+             </div>
+             <div class="icons">
+        <ul>
+          <li><img src="./image/image6.png" alt="twitter"></li>
+          <li><img src="./image/image7.png" alt="twitter"></li>
+          <li><img src="./image/image8.png" alt="twitter"></li>
+           <li><img src="./image/image9.png" alt="twitter"></li>
+          <li><img src="./image/imge5.png" alt="twitter"></li>
+        </ul>
+        </div> 
+          <button type="button">Get My Resume</button>
+
+                
+                <div class="end">
+                 <ul class="photo">
```

Comment:
- [Optional] Dear @ismailmunyentwari9, One major thing missing here is the margins (top, bottom, left, right) defined in the design. Please try to address this. This is only optional because it has not been mentioned to you. 
- [Optional] Another thing is your fonts. Please make sure to use a `Poppins` font type as defined in the Figma template.
### Figma design
![figma](https://user-images.githubusercontent.com/69077061/205422354-cae1cfb0-87e1-4984-a33d-76d1d4289f73.PNG)


### Your design on `375px` screen
![your](https://user-images.githubusercontent.com/69077061/205422338-42fdf516-2a50-4e31-969c-04d42aaa60c1.PNG)